src/sentry/incidents/logic.py

```diff
@@ -610,21 +617,15 @@ def create_alert_rule(
             alert_threshold=alert_threshold,
             resolve_threshold=resolve_threshold,
             threshold_period=threshold_period,
+            include_all_projects=include_all_projects,
         )
-        subscriptions = bulk_create_snuba_subscriptions(
-            projects,
-            tasks.INCIDENTS_SNUBA_SUBSCRIPTION_TYPE,
-            dataset,
-            query,
-            aggregation,
-            time_window,
-            resolution,
-        )
-        subscription_links = [
-            AlertRuleQuerySubscription(query_subscription=subscription, alert_rule=alert_rule)
-            for subscription in subscriptions
-        ]
-        AlertRuleQuerySubscription.objects.bulk_create(subscription_links)
+        if include_all_projects:
```

Comment:
What should the API do if `include_all_projects` and a safe list of projects are sent at the same time?

Reply:
My thought is that `include_all_projects` beats `projects`, and so the projects will be ignored. I can also just validate against it and return an error if both are passed.